2-data-model/data-modeling-in-rethinkdb.md

```diff
@@ -65,9 +65,7 @@ __Advantages of using embedded arrays:__
 
 __Disadvantages of using embedded arrays:__
 
-- Any operation on an author document requires loading all posts into
-  memory. Any update to the document requires rewriting the full array
-  to disk.
+- Any operation on a document in the `author` table requires loading all posts into memory. As well, any update to the document requires rewriting the entire document to disk. At the moment, RethinkDB does not support modifying only the single value on disk. 
```

Comment:
"requires loading all posts into memory" can be misunderstood I think. It should be "requires loading all posts by the given author into memory"